README.md

```diff
@@ -1,2 +1,220 @@
-# world-covid-19-data
-This web app provides current and historical Covid-19 data for every country in the world. Available data includes confirmed case counts and deaths, and is updated everyday.
+# Global Covid Statistics
+
+<a name="readme-top"></a>
+
+<div align="center">
+  
+  <h3><b>Cryptocurrencies-Metrics Readme</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Global Covid Statistics <a name="about-project"></a>
+
+
+
+**Global Covid Statistics**  is a web application the gives the total number of Covid-19 cases per country. It is created using the coinstats API. Countries and/or regions are displayed on the homepage. The figures are displayed when a country is clicked on the homepage.
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML, CSS, JavaScript, React , API, Redux</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://localhost:8080/">Dev Server</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Package Manager</summary>
+  <ul>
+    <li><a href="#">npm</a></li>
+  </ul>
+</details>
+<details>
+  <summary>APIs</summary>
+  <ul>
+      <li><a href="'https://api.coinstats.app/public/v1/coins'">Coinstats API</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **User clicks on a country**
+- **[User can get more details about the selected country]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- You can see how it looks like [here](https://global-covid-statistics.onrender.com/)
+- Presentation of the project is [here](https://www.loom.com/share/a25ce530e2084af98bb51998c4111f76)
```

Comment:
- [ ] Great job working on the video presentation but I think it would be great if you  highlight some interesting piece of code or something you built that you are very proud of. since it is mentioned in the requirements and it is important to lnow what you have learned as new feature or unique thing you want to highlight ⭐💪🏻🚀🔥 so please record another video with mentioning that 👍🏻

Reply:
I did mention the spinner. That is something new that I learned using react bootstrap. I admit I wasn't explicit enough and will do another video. Thanks for the review.

Reply:
Hello @dazehinn I noticed that you mentioned it and great job but this is extra feature what I mean to highly a piece of code you are proud to do it 🙏🏻🔥🚀

Reply:
- [ ] It would be better to highlight a **piece of code** you are proud of. Meaning you will show us in your code editor the spinner code you added and not the browser as you have shown in the presentation. I hope this is clearer now